source/cloud/azure/azureml.md

```diff
@@ -102,7 +101,9 @@ ml_client = MLClient.from_config(
 
 You will need to create a [compute target](https://learn.microsoft.com/en-us/azure/machine-learning/concept-compute-target?view=azureml-api-2#azure-machine-learning-compute-managed) using Azure ML managed compute ([AmlCompute](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-ml/0.1.0b4/azure.ai.ml.entities.html)) for remote training.
 
-Note: Be sure to check limits within your available region.
+```{note}
```

Comment:
This was marked as note, but it's unclear where do we check this? I wasn't able to find where to check this

Reply:
I think this is the quotas we were looking at the other day. Maybe rewording with "quota" would make this more understandable. Or if this is just duplication of other advice we could remove it.